middle_end/flambda/basic/exn_continuation.ml

```diff
@@ -122,14 +122,17 @@ let without_extra_args t =
 let all_ids_for_export t =
```

Comment:
I think this would probably be better with: `let_all_ids_for_export { exn_handler; extra_args; } = ...` so we will get a compilation error here if we add another field to the record.

Reply:
Makes sense. Changing `import` too while I'm at it.